fix(minidump): Do not cutoff `ul` in wizard html (#1029)
getsentry/sentry-docs: src/collections/_documentation/platforms/minidump/index.md

<!-- WIZARD -->
## Platform and Language Support

> Minidumps are currently supported for **Windows, macOS and Linux**. There is no limitation as to which programming language can be used. Sentry can also demangle symbols from the following languages:

Reviewer: ```suggestion
Sentry currently supports minidumps for **Windows, macOS and Linux**. There is no limitation as to which programming language you can use. Sentry can also demangle symbols from the following languages:
```

-   C and C++
-   ObjectiveC and ObjectiveC++
-   Swift
-   Rust

> Other languages will show the mangled name instead.

Reviewer: ```suggestion
These other languages will show the mangled name instead.
```

Author: Actually it is saying languages *not* in the list above

Reviewer: Aha. Okay, made a new suggestion.
